File: src/main/java/plugin/psi/VardefSubtree.java
package plugin.psi;

import com.intellij.lang.ASTNode;
import com.intellij.psi.tree.IElementType;
import org.antlr.intellij.adaptor.psi.IdentifierDefSubtree;
import org.jetbrains.annotations.NotNull;

public class VardefSubtree extends IdentifierDefSubtree {

	public VardefSubtree(@NotNull ASTNode node, @NotNull IElementType idElementTyp) {
		super(node, idElementTyp);
	}
}

File: src/main/kotlin/plugin/psi/VardefSubtree.kt
package plugin.psi

import com.intellij.lang.ASTNode
import com.intellij.psi.tree.IElementType
import org.antlr.intellij.adaptor.psi.IdentifierDefSubtree

open class VardefSubtree(node: ASTNode, idElementTyp: IElementType) : IdentifierDefSubtree(node, idElementTyp)
